data/jibauni/rebel jibauni 0 consultations.txt

```diff
@@ -0,0 +1,359 @@
+		has "rebel jibauni: will transport upon request"
+		not "jibauni: disliked by the rebels"
+	on offer
+		conversation
+			`A Rebel Jibauni comes up and announces something. Your translator converts it to, "Take me to <planet> for our leaders and the good of the Rebel Jibauni. Do not dare refuse. This has been 'Requested by the Real Jibauni'."`
+				accept
+	on complete
+		event "rebel jibauni: first transportation done and second ready to offer" 2 14
+		conversation
+			`The Rebel Jibauni thanks you, and tells you that you will be on call whenever the leaders want.`
+
+event "rebel jibauni: first transportation done and second ready to offer"
+
+mission "Rebel Jibauni: Government Transportation 2"
+	landing
+	source Andwerfen
+	destination Minmanmun
+	passengers 1
+	to offer
+		has "event: rebel jibauni: first transportation done and second ready to offer"
+	on offer
+		conversation
+			`A Rebel Jibauni comes up to the <ship> and says, "The Rebel Jibauni leaders request you to kindly transport me to <planet>. They will back it up. This has been 'Requested by the Real Jibauni'."`
+				accept
+	on complete
+		event "rebel jibauni: second transportation done and third ready to offer" 3 9
+		conversation
+			`The Rebel Jibauni leaves after telling you that you will be on call as needed.`
+
+event "rebel jibauni: second transportation done and third ready to offer"
+
+mission "Rebel Jibauni: Government Transportation 3"
+	landing
+	source Andwerfen
+	destination Muntisir
+	passengers 3
+	to offer
+		has "event: rebel jibauni: second transportation done and third ready to offer"
+	on offer
+		conversation
+			`A group of Rebels approach the landing pad that the <ship> is landed on and ask you to take them to <planet>. You do not seem to be able to refuse, as they tell you that this transport mission has been "Requested by the Real Jibauni".`
+				accept
+	on complete
+		event "rebel jibauni: government transportation done" 10 25
+		payment 50000
+		conversation
+			`The group disembarks, but one of them gives you <payment>, "As a small token of our gratitude."`
+
+event "rebel jibauni: government transportation done"
+
+mission "Rebel Jibauni: Bring Them Back"
+	name "Transport Rebels Back"
+	description "Get the Rebel Jibauni that you transported out to <stopovers> back to <destination>."
+	source Andwerfen
+	stopover Minmanmun
+	stopover Muntisir
+	stopover Sinsansun
+	passengers 5
+	to offer
+		has "event: rebel jibauni: government transportation done"
+	on offer
+		payment 10000
+		conversation
+			`The spaceport of <origin> is a very busy place. As you walk around admiring the sights, you are suddenly pulled into an alley by an unknown assailant and yanked away from the main thoroughfare. You are then knocked out.`
+			`	A while later, when you wake up, you find yourself in the same place as you were when the Rebel Jibauni Leaders spoke to you. As you think this, the same Jibauni enter and are seated.`
```

Comment:
There's inconsistent capitalization of **L**eaders in this mission chain. I'd lean towards lower case, but it doesn't really matter so long as it's consistent.